docs/tutorials/config-examples.md

```diff
@@ -0,0 +1,514 @@
+---
+id: config-examples
+sidebar_position: 12
+title: Configuration Examples
+---
+
+## Ready-to-Use Configuration Examples
+
+Setting up cross-seed correctly can be challenging, especially when dealing with
+JavaScript syntax and various environment considerations. This guide provides
+ready-to-use configuration examples for various common scenarios.
+
+### Basic Configuration Syntax
+
+Every cross-seed config.js file follows this basic structure:
+
+```js
+module.exports = {
+	// Your settings go here
+	setting1: "value",
+	setting2: ["array", "of", "values"],
+	setting3: true,
+};
+```
+
+:::caution JavaScript Syntax Remember these essential JavaScript syntax rules:
```

Comment:
always need a double new line between :::caution and the content of the admonition because otherwise Prettier will format them onto the same line and it will break rendering. same happens at the end of this admonition